### ISSUE TITLE
Custom attributes weren't sent

### ISSUE THREAD

**Issue by vani2** [DataDog/dd-sdk-ios#1601]
DataDog SDK version: 2.5.0, installed via SPM.
Xcode 15.1 runs on iPhone 14 Pro, and iOS 17.2.1.

I want to set custom global attributes, [docs describe it here](https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios/?tab=swift#track-custom-global-attributes).

But I don't see it in the session. 
I'm doing something like this

```swift
RUMMonitor.shared().addAttribute(forKey: "key1", value: "value1")
```
There's no issues with adding user info, for instance.

```swift
Datadog.setUserInfo(email: email)
```

Basically, what I want to achieve is the same as I have with my other React native app on the screenshot.

![Screenshot 2023-12-20 at 15 19 54](https://github.com/DataDog/dd-sdk-ios/assets/4734983/cd3daa99-15ae-4e0a-86f9-a89b33788e27)

I don't see that section completely in my iOS app sessions.

**Comment by ganeshnj** [comment]
I tested in Example here in the SDK on the latest release and I can see the custom attributes

```diff
@@ -73,6 +73,7 @@ class ExampleAppDelegate: UIResponder, UIApplicationDelegate {
             )
         )
         RUMMonitor.shared().debug = true
+        RUMMonitor.shared().addAttribute(forKey: "key1", value: "value1")

         // Create Logger
         logger = Logger.create(
```

![image](https://github.com/DataDog/dd-sdk-ios/assets/8882380/e2e58296-cdf1-4e6b-b0ab-ad39d051ba58)

Do you have an example application/code which we can use to reproduce the issue?

**Comment by ganeshnj** [comment]
No activity, closing because working as expected.
